var/spack/repos/builtin/packages/ninja-fortran/package.py

```diff
@@ -23,27 +23,36 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
-import os
 
 
 class NinjaFortran(Package):
-    """ A Fortran capable fork of ninja """
+    """A Fortran capable fork of ninja."""
+
     homepage = "https://github.com/Kitware/ninja"
     url      = "https://github.com/Kitware/ninja/archive/v1.7.2.gcc0ea.kitware.dyndep-1.tar.gz"
 
     version('1.7.2', '3982f508c415c0abaca34cb5e92e711a')
 
-    extends('python')
+    depends_on('python', type=('build', 'run'))
```

Comment:
Doesn't build any Python libraries, therefore `depends_on` instead of `extends`.